x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_expression.test.tsx

```diff
@@ -69,7 +69,10 @@ describe('xy_expression', () => {
         position: Position.Left,
       };
 
-      expect(legendConfig.fn(null, args, {})).toEqual({
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const result = legendConfig.fn(null, args, {} as any);
```

Comment:
Would be nice to make the `{}` optional

Reply:
I'm not sure we can make it optional as then we would need to go through all expression functions in OSS, Lens and Canvas (and that would be a breaking change) to make those functions support potentially missing third argument. Unless I'm missing something, happy to hear a good solution to this.

Reply:
We could provide a mock helper to create a dummy execution context, then we don't have to overwrite all the type safety.
```ts
      const result = legendConfig.fn(null, args, createMockExecutionContext());
```

Reply:
Done. I've added `createMockExecutionContext()` helper.